qp_shotgun/qc_trim/qc_trim.py

```diff
@@ -219,14 +206,6 @@ def _per_sample_ainfo(out_dir, samples, fwd_and_rev=False):
         # was kept after quality control and filtering for host data
         raise ValueError("No sequences left after running KneadData")
 
-    # Generate the missing files
```

Comment:
We shouldn't need to generate missing files now... but want to check in on the edge case of very low read # input files to test this.